### PR TITLE
Url config minimal patch

### PR DESCRIPTION
**Pull request by JoranAngevaare**
Follow https://github.com/XENONnT/straxen/pull/1079

### CODE REVIEW

File: wfsim/strax_interface.py [hunk 1]
```diff
@@ -762,8 +768,8 @@ def set_config(self,):
             if overrides is not None:
                 self.config_nveto.update(overrides)
 
-            self.to_pe_nveto = straxen.get_correction_from_cmt(self.run_id,
-                               self.config['gain_model_nv'])
+            to_pe_nv = self.gain_model_nv
+            self.to_pe_nveto = straxen.get_correction_from_cmt(self.run_id, to_pe_nv)
```
Comment (jmosbacher) [inline wfsim/strax_interface.py:772]: Is this left here on purpose?

**Review by jmosbacher — COMMENTED** [review]
Looks good, thanks for doing this! just wondering if nveto to_pe is some special case here?

File: wfsim/strax_interface.py (hunk 1 above)
Comment (JoranAngevaare) [reply wfsim/strax_interface.py]: whoops, thanks Yossi!

**Review by jmosbacher — APPROVED** [review]
A good example of "don't let perfect be the enemy of good". It solves the problem :)

**Comment by jmosbacher** [comment]
@JoranAngevaare looks like the test is the issue [here](https://github.com/XENONnT/WFSim/blob/0f83fc14fd3c8c377da83237e97a355acabacf1b/tests/test_wfsim.py#L207)

**Comment by JoranAngevaare** [comment]
thx 👍 !

**Comment by JoranAngevaare** [comment]
I'll remove the scaffolding and merge after straxen 2.0.0 is released

**Pull request merged**